mypy/types.py

```diff
@@ -1741,7 +1747,9 @@ def set_typ_args(tp: Type, new_args: List[Type], line: int = -1, column: int = -
     if isinstance(tp, Instance):
         return Instance(tp.type, new_args, line, column)
     if isinstance(tp, TupleType):
-        return tp.copy_modified(items=new_args)
+        fallback_args = [UnionType.make_simplified_union(new_args)]
```

Comment:
Ok, I removed this as well. How would we remember to add this logic and the one [here](https://github.com/python/mypy/pull/3129#discussion_r123881917) (both of which I removed) to a new PR in the future?